### ISSUE TITLE
Regarding https://www.enterprisedb.com/docs/

### ISSUE THREAD

**Issue by kelpoole** [EnterpriseDB/docs#1109]
## Summary
Hadoop Data Adapter and Mongo Data Adapter should be titled Hadoop Foreign Data Wrapper and Mongo Foreign Data Wrapper to correspond to the title so the documents when the link is chosen.  It would also be nice if the we could have all Foreign Data Wrappers together (under a subheading) - MySQL, Hadoop, and Mongo and if we could have the Connectors together - JDBC, ODBC, .net, and OCL

## Where did you see the problem?

Docs main menu

**Comment by kelpoole** [comment]
If subheadings can't be used then maybe we should consider dividing Integration into multiple topics instead - pgPool and PgBouncer 

Foreign Data Wrappers
     Mongo
     MySQL
     Hadoop

Connectors
    JDBC
    ODBC
    OCL
    .NET

**Comment by cero-miedo** [comment]
![image](https://user-images.githubusercontent.com/62722976/112478204-abe71f80-8d6b-11eb-9ed2-1cbf8de4dcde.png)

I've started a branch to address the issues raised here. One thing I've noticed is that the documentation URL slugs has been inherited from source files, so all Foreign Data Wrappers do not have the word _wrapper_ in the slug. 

We can add this, however it would have some knock-on implications like manually updating redirects. I'm going to mark this issue as solved by the upcoming PR, however if the slug issue is something that needs action, we should work off a new issue.
